docs/pages/admin-guides/access-controls/access-requests/resource-requests.mdx

```diff
@@ -406,6 +406,72 @@ Requesting access to a Kubernetes Namespace allows you to access all resources
 in that namespace but you won't be able to access any other supported resources
 in the cluster.
 
+##### Enforce users to request only allowed Kubernetes Resources
+
+The `request.kubernetes_resources` field allows you to restrict what kinds of Kubernetes 
+resources a user can access request to.
+
+The following role assigned to a user, restricts access requesting to only Kubernetes namespaces.
+If a user attempts to make a request to Kubernetes resources other than `namespace` then the request
+will be rejected:
+
+```yaml
+kind: role
+metadata:
+  name: requester-kube-access
+version: v7
+spec:
+  allow:
+    request:
+      search_as_roles:
+        - "kube-access"
+      kubernetes_resources:
+        - kind: "namespace"
+  deny: {}
+```
+
+The following role assigned to a user, restricts access requesting to only Kubernetes namespaces and/or pods.
+
+```yaml
+kind: role
+metadata:
+  name: requester-kube-access
+version: v7
+spec:
+  allow:
+    request:
+      search_as_roles:
+        - "kube-access"
+      kubernetes_resources:
+        - kind: "namespace"
+        - kind: "pod"
+  deny: {}
+```
+
+The following role assigned to a user, restricts users from access requesting to a Kubernetes cluster but
+allows requesting to any Kubernetes resources:
```

Comment:
```suggestion
The following role, when assigned to a user, allows them to request access to any specific
Kubernetes resource, but does *not* allow requesting access to the entire cluster.
```

Reply:
i removed the mention of not allowing requesting access to the entire cluster here, and moved it towards the beginning of the section, see comment: https://github.com/gravitational/teleport/pull/48480/files#r1830229498

Reply:
btw, you and zac both gave review around same time i think, i took some of zac's suggestion over yours (or tried to combine the best parts)